crates/router/src/routes/payment_link.rs

```diff
@@ -92,6 +99,7 @@ pub async fn initiate_secure_payment_link(
     let payload = api_models::payments::PaymentLinkInitiateRequest {
         payment_id,
         merchant_id: merchant_id.clone(),
+        locale: None,
```

Comment:
is the locale only for open payment links, and not iframe embeded links?

Reply:
`PaymentLinkRequest` is used for both payment links and secure payment links. In the case of payment links and status routes, we pass the locale to the flow. However, secure links already pass headers to the `initiate_secure_payment_link_flow` function. So I was consuming the locale from headers in secure_payment_link, and in all other cases, I am not passing headers to flows; instead, I am directly passing the locale.

Reply:
can we unify it, it looks confusing at first glance!